src/test/resources/modelwords.txt

```diff
@@ -741,6 +741,7 @@ Cust
 cust
 Custer
 custpop
+customizations
```

Comment:
Agreed. Through the years, I've also caught some genuine spelling errors that where mistakenly white-listed here... I suspect there might be a few more of those in there...